README.md

```diff
@@ -306,3 +317,8 @@ Blogs:
 - [Building a Large-scale Distributed Storage System Based on Raft](https://www.cncf.io/blog/2019/11/04/building-a-large-scale-distributed-storage-system-based-on-raft/), by Ed Huang
 
 ## Benchmark & Testing
+
+## Build A Database from Scratch
+- [Let's Build a Simple Database](https://cstack.github.io/db_tutorial/)
+- [builddatabase](https://github.com/ngaut/builddatabase)
```

Comment:
most content of this repo is empty, we can add this material when it's finished.